app/forms/candidate_interface/continuous_applications/which_course_are_you_applying_to_step.rb

```diff
@@ -5,6 +5,8 @@ class WhichCourseAreYouApplyingToStep < DfE::WizardStep
       attr_accessor :provider_id, :course_id
       validates :provider_id, :course_id, presence: true
 
+      validate CandidateInterface::ContinuousApplications::CourseChoiceValidator.new, on: :course_choice
```

Comment:
If you move the `CourseChoiceValidator` to `app/validators` you could replace this with `      validates self, course_choice: true, on: :course_choice`

Reply:
Move it to app/validators